functional/android/build.gradle

```diff
@@ -0,0 +1,17 @@
+plugins {
+    id 'org.jetbrains.kotlin.jvm'
+    id 'org.jmailen.kotlinter'
+    id 'java-gradle-plugin'
+}
+
+repositories {
+    jcenter()
+    google()
+}
+
+dependencies {
+    runtimeOnly project(':')
+    runtimeOnly 'com.android.tools.build:gradle:3.5.3'
+    runtimeOnly 'org.jetbrains.kotlin:kotlin-gradle-plugin'
```

Comment:
I could be using `testRuntimeOnly` but adding it to `runtimeOnly` make it gets picked by `java-gradle-plugin`'s `pluginUnderTestMetadata` task and automatically applied to TestKit invoked projects.